src/main/java/mfarr/backendspringbootrelationsuitwerkingen/models/TelevisionWallBracket.java

```diff
@@ -0,0 +1,49 @@
+package mfarr.backendspringbootrelationsuitwerkingen.models;
+
+
+import jakarta.persistence.*;
+
+@Entity
+public class TelevisionWallBracket {
```

Comment:
Deze klasse is nu uit de uitwerking gehaald. Je kunt daar nog eens naar kijken als je wilt.